engine/logic_monad.ml

```diff
@@ -107,7 +107,7 @@ struct
   let print_debug   s = make (fun _ -> Feedback.msg_info s)
   let print_info    s = make (fun _ -> Feedback.msg_info s)
   let print_warning s = make (fun _ -> Feedback.msg_warning s)
-  let print_error   s = make (fun _ -> Feedback.msg_error s)
+  (* let print_error   s = make (fun _ -> Feedback.msg_error s) *)
```

Comment:
Why comment instead of removing?

Reply:
Yup, it seems that `print_error` actually makes little sense, instead you want to "raise an error"